connectors/citrus-openapi/src/main/java/org/citrusframework/openapi/model/v3/Oas30ModelHelper.java

```diff
@@ -219,7 +220,7 @@ public static Map<String, OasSchema> getRequiredHeaders(Oas30Response response)
 
         return response.headers.entrySet()
                 .stream()
-                .filter(entry -> entry.getValue().required)
+                .filter(entry -> Boolean.TRUE.equals(entry.getValue().required))
```

Comment:
Since this is part of a public API that can be easily accessed within the “Oas30ModelHelper”, it seems that it would be easily testable.

Reply:
@bbortt - I have created a test, please check.

Reply:
awesome, many thanks! that proves the desired behavior quit well 🚀